src/components/schedule-inputs.tsx

```diff
@@ -32,8 +53,8 @@ export function ScheduleInputs(props: ScheduleInputsProps): JSX.Element | null {
 
   let cronString;
   try {
-    if (props.schedule !== undefined && !props.errors['schedule']) {
-      cronString = cronstrue.toString(props.schedule);
+    if (props.model.schedule !== undefined && !props.errors['schedule']) {
```

Comment:
Nit: here and everywhere in PR just `props.model.schedule` assertion can be used instead of `props.schedule !== undefined`. I do see an argument for better descriptiveness / more clarity in using `props.schedule !== undefined`

Reply:
Ok I see how you would want to use explicit `!== undefined` assertions taking in account there are a lot of time / number-based assertions where something like '00000' would be false

Reply:
The case I wanted to exclude was `props.model.schedule === ''`, which suggests that the schedule isn't specified. In practice, though, since we now default to "weekday" (`0 0 * * MON-FRI`) the schedule shouldn't be blank.